sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2Suite.scala

```diff
@@ -375,7 +375,8 @@ class DataSourceV2Suite extends QueryTest with SharedSQLContext {
     }
   }
 
-  test("SPARK-25700: do not read schema when writing in other modes except append and overwrite") {
+  test("SPARK-25700: do not read schema when writing in other modes" +
+    " except append and overwrite") {
```

Comment:
Ur, do we need to break the test title? For me, it looks like exactly same and this pass ScalaStyle before, doesn't it?